master/cluster.go

```diff
@@ -992,7 +993,44 @@ func (c *Cluster) updateMetaNodeBaseInfo(nodeAddr string, id uint64) (err error)
 	return
 }
 
-func (c *Cluster) addMetaNode(nodeAddr, zoneName string, nodesetId uint64) (id uint64, err error) {
+// RaftPartitionCanUsingDifferentPortEnabled check whether raftPartitionCanUsingDifferentPort param become effective
+// raftPartitionCanUsingDifferentPort param take into force only when
+//  1. raftPartitionCanUsingDifferentPort set true,
+//  2. all data nodes and meta nodes are registered with HeartbeatPort and ReplicaPort
+func (c *Cluster) RaftPartitionCanUsingDifferentPortEnabled() bool {
+	if !c.cfg.raftPartitionCanUsingDifferentPort {
```

Comment:
The realization of 'CheckOrStoreConstCfg' uses a special file named 'constcfg'. Why not store it into the rocksdb storage, which is a common way to store metadata of the master? Besides, there's no need to make the judgment and loop through all the nodes. Once the result in the rocksdb is set, it can't be changed, and it can be used directly by the judge.